satpy/modifiers/angles.py

```diff
@@ -231,11 +269,52 @@ def _sanitize_observer_look_args(*args):
         elif isinstance(arg, (float, np.float64, np.float32)):
             # round floating point numbers to nearest tenth
             new_args.append(round(arg, 1))
+        elif _is_chunk_tuple(arg) and _chunks_are_irregular(arg):
+            new_chunks = _regular_chunks_from_irregular_chunks(arg)
+            new_args.append(new_chunks)
         else:
             new_args.append(arg)
     return new_args
 
 
+def _sanitize_args_with_chunks(*args):
+    new_args = []
+    for arg in args:
+        if _is_chunk_tuple(arg) and _chunks_are_irregular(arg):
+            new_chunks = _regular_chunks_from_irregular_chunks(arg)
+            new_args.append(new_chunks)
+        else:
+            new_args.append(arg)
+    return new_args
+
+
+def _is_chunk_tuple(some_obj: Any) -> bool:
+    if not isinstance(some_obj, tuple):
+        return False
+    if not all(isinstance(sub_obj, tuple) for sub_obj in some_obj):
+        return False
```

Comment:
Is this line tested?

Reply:
No and it isn't possible right now because all the calling functions of the cached functions provide cache tuples and no other tuples. I could make a fake function to cache and test these cases...let me try that.

Reply:
if this code is unreachable anyway, we could just remove it then?